jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/data/TFRecordBuilders.scala

```diff
@@ -111,7 +133,7 @@ class SequenceExampleBuilder[C, S](contextFeaturesPreprocessor: FeaturePreproces
     * @param featureValues
     * @return
```

Comment:
Should we completes the missing return doc in this file? Or remove the empty @return if it's obvious?